app/src/main/java/com/mapbox/maps/testapp/examples/RestrictBoundsActivity.kt

```diff
@@ -37,7 +38,7 @@ class RestrictBoundsActivity : AppCompatActivity() {
     mapboxMap.loadStyle(
       style(Style.SATELLITE_STREETS) {
         +geoJsonSource(BOUNDS_ID) {
-          data("")
+          featureCollection(FeatureCollection.fromFeatures(listOf()))
```

Comment:
If this works, let's move the code to the annotation plugin, but this will break some other examples as well. For example: RestrictBoundsActivity.